docs/program/python-venv.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Prepare your Python Virtual Environment"
+linkTitle: "Prepare your Python Virtualenv for Viam"
+weight: 50
+type: "docs"
+description: "Prepare your Python Virtual Environment to program machines with the Python SDK."
+images: ["/services/icons/sdk.svg"]
+tags: ["client", "sdk", "application", "sdk", "fleet", "program", "python", "venv"]
+---
+
+To manage Python packages for your Viam application, it is recommended that you use a virtual environment, or `venv`.
+By using a `venv`, you can install Python packages like Viam's client SDK just within the virtual environment, and not globally, which could cause conflicts with other projects or cause issues within your system.
+
+Follow this guide to set up a fresh virtual environment on your working computer and install the Python SDK as a requirement for your Viam client application.
+
+## Setup your project
+
+First, create a directory to house your project.
```

Comment:
that might be a bit of an anthropomorphisation we can get rid of for readability:
```suggestion
First, create a directory for your project.
```